requirements/pip_tools.txt

```diff
@@ -8,10 +8,13 @@ click==8.0.1
     # via pip-tools
 pep517==0.10.0
     # via pip-tools
-pip-tools==6.1.0
+pip-tools==6.2.0
     # via -r requirements/pip_tools.in
 toml==0.10.2
     # via pep517
+wheel==0.36.2
```

Comment:
@Jawayria I think you might know, why is this adding `wheel`?

Reply:
wheel is required by pip-tools so pip-tools added it